websearch java.lang.IllegalAccessError

### JabRef version

Other (please describe below)

### Operating system

GNU / Linux

### Details on version and operating system

JabRef 5.6--2022-03-10--749b6b2 Linux 5.16.12-200.fc35.x86_64 amd64  Java 17.0.2  JavaFX 17.0.2-ea+3

### Checked with the latest development build

- [X] I made a backup of my libraries before testing the latest development version.
- [X] I have tested the latest development version and the problem persists

### Steps to reproduce the behaviour

1. Websearch for `critical, engaged and change-oriented`

it takes some seconds, and then the search results window shows the error

### Appendix

...
<details>

<summary>Log File</summary>

```
java.lang.Exception: java.lang.IllegalAccessError
	at org.jabref@5.6.76/org.jabref.gui.util.DefaultTaskExecutor.convertToException(Unknown Source)
	at org.jabref@5.6.76/org.jabref.gui.util.DefaultTaskExecutor.lambda$getJavaFXTask$5(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.CompositeEventHandler.dispatchBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventUtil.fireEventImpl(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventUtil.fireEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.event.Event.fireEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.concurrent.EventHelper.fireEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.concurrent.Task.fireEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.concurrent.Task.setState(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.concurrent.Task$TaskCallable.lambda$call$2(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.application.PlatformImpl.lambda$runLater$10(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.application.PlatformImpl.lambda$runLater$11(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.glass.ui.InvokeLaterDispatcher$Future.run(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.glass.ui.gtk.GtkApplication.enterNestedEventLoopImpl(Native Method)
	at org.jabref.merged.module@5.6.76/com.sun.glass.ui.gtk.GtkApplication._enterNestedEventLoop(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.glass.ui.Application.enterNestedEventLoop(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.glass.ui.EventLoop.enter(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.tk.quantum.QuantumToolkit.enterNestedEventLoop(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.stage.Stage.showAndWait(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.scene.control.HeavyweightDialog.showAndWait(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.scene.control.Dialog.showAndWait(Unknown Source)
	at org.jabref@5.6.76/org.jabref.gui.JabRefDialogService.showCustomDialogAndWait(Unknown Source)
	at org.jabref@5.6.76/org.jabref.gui.importer.fetcher.WebSearchPaneViewModel.search(Unknown Source)
	at org.jabref@5.6.76/org.jabref.gui.importer.fetcher.WebSearchPaneView.lambda$initialize$2(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.CompositeEventHandler.dispatchBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.CompositeEventDispatcher.dispatchBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventUtil.fireEventImpl(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventUtil.fireEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.event.Event.fireEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.scene.Scene$KeyHandler.process(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.scene.Scene.processKeyEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.scene.Scene$ScenePeerListener.keyEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.tk.quantum.GlassViewEventHandler$KeyEventNotification.run(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.tk.quantum.GlassViewEventHandler$KeyEventNotification.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.tk.quantum.GlassViewEventHandler.lambda$handleKeyEvent$1(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.tk.quantum.QuantumToolkit.runWithoutRenderLock(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.tk.quantum.GlassViewEventHandler.handleKeyEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.glass.ui.View.handleKeyEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.glass.ui.View.notifyKey(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.glass.ui.gtk.GtkApplication._runLoop(Native Method)
	at org.jabref.merged.module@5.6.76/com.sun.glass.ui.gtk.GtkApplication.lambda$runLoop$11(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalAccessError
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Unknown Source)
	at java.base/java.lang.reflect.Constructor.newInstance(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinTask.getThrowableException(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinTask.reportException(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinTask.invoke(Unknown Source)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateParallel(Unknown Source)
	at java.base/java.util.stream.AbstractPipeline.evaluate(Unknown Source)
	at java.base/java.util.stream.ReferencePipeline.collect(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.importer.fetcher.CompositeSearchBasedFetcher.performSearch(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.importer.SearchBasedFetcher.performSearch(Unknown Source)
	at org.jabref@5.6.76/org.jabref.gui.importer.fetcher.WebSearchPaneViewModel.lambda$search$3(Unknown Source)
	at org.jabref@5.6.76/org.jabref.gui.util.BackgroundTask$1.call(Unknown Source)
	at org.jabref@5.6.76/org.jabref.gui.util.DefaultTaskExecutor$1.call(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.concurrent.Task$TaskCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	... 1 more
Caused by: java.lang.IllegalAccessError: superclass access check failed: class org.glassfish.jaxb.runtime.api.JAXBRIContext (in module org.jabref.merged.module) cannot access class jakarta.xml.bind.JAXBContext (in module jakarta.xml.bind) because module org.jabref.merged.module does not read module jakarta.xml.bind
	at java.base/java.lang.ClassLoader.defineClass1(Native Method)
	at java.base/java.lang.ClassLoader.defineClass(Unknown Source)
	at java.base/java.lang.ClassLoader.defineClass(Unknown Source)
	at java.base/java.security.SecureClassLoader.defineClass(Unknown Source)
	at java.base/jdk.internal.loader.BuiltinClassLoader.defineClass(Unknown Source)
	at java.base/jdk.internal.loader.BuiltinClassLoader.findClassInModuleOrNull(Unknown Source)
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClassOrNull(Unknown Source)
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(Unknown Source)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(Unknown Source)
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
	at java.base/java.lang.Class.getDeclaredMethods0(Native Method)
	at java.base/java.lang.Class.privateGetDeclaredMethods(Unknown Source)
	at java.base/java.lang.Class.getMethodsRecursive(Unknown Source)
	at java.base/java.lang.Class.getMethod0(Unknown Source)
	at java.base/java.lang.Class.getMethod(Unknown Source)
	at jakarta.xml.bind@3.0.1/jakarta.xml.bind.ContextFinder.newInstance(Unknown Source)
	at jakarta.xml.bind@3.0.1/jakarta.xml.bind.ContextFinder.newInstance(Unknown Source)
	at jakarta.xml.bind@3.0.1/jakarta.xml.bind.ContextFinder.find(Unknown Source)
	at jakarta.xml.bind@3.0.1/jakarta.xml.bind.JAXBContext.newInstance(Unknown Source)
	at jakarta.xml.bind@3.0.1/jakarta.xml.bind.JAXBContext.newInstance(Unknown Source)
	at jakarta.xml.bind@3.0.1/jakarta.xml.bind.JAXBContext.newInstance(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.importer.fileformat.MedlineImporter.initUmarshaller(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.importer.fileformat.MedlineImporter.unmarshallRoot(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.importer.fileformat.MedlineImporter.importDatabase(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.importer.fetcher.MedlineFetcher.fetchMedline(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.importer.fetcher.MedlineFetcher.performSearch(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.importer.fetcher.CompositeSearchBasedFetcher.lambda$performSearch$1(Unknown Source)
	at java.base/java.util.stream.ReferencePipeline$7$1.accept(Unknown Source)
	at java.base/java.util.HashMap$KeySpliterator.tryAdvance(Unknown Source)
	at java.base/java.util.stream.StreamSpliterators$WrappingSpliterator.lambda$initPartialTraversalState$0(Unknown Source)
	at java.base/java.util.stream.StreamSpliterators$AbstractWrappingSpliterator.fillBuffer(Unknown Source)
	at java.base/java.util.stream.StreamSpliterators$AbstractWrappingSpliterator.doAdvance(Unknown Source)
	at java.base/java.util.stream.StreamSpliterators$WrappingSpliterator.tryAdvance(Unknown Source)
	at java.base/java.util.stream.StreamSpliterators$UnorderedSliceSpliterator$OfRef.forEachRemaining(Unknown Source)
	at java.base/java.util.stream.AbstractPipeline.copyInto(Unknown Source)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
	at java.base/java.util.stream.ReduceOps$ReduceTask.doLeaf(Unknown Source)
	at java.base/java.util.stream.ReduceOps$ReduceTask.doLeaf(Unknown Source)
	at java.base/java.util.stream.AbstractTask.compute(Unknown Source)
	at java.base/java.util.concurrent.CountedCompleter.exec(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinPool.scan(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(Unknown Source)

```

</details>

Caused by: java.lang.IllegalAccessError: superclass access check failed: class org.glassfish.jaxb.runtime.api.JAXBRIContext (in module org.jabref.merged.module) cannot access class jakarta.xml.bind.JAXBContext (in module jakarta.xml.bind) because module org.jabref.merged.module does not read module jakarta.xml.bind

Related to #8517

The following error message emerges when using `SearchALL`.

```
java.lang.Exception: java.lang.IllegalAccessError
	at org.jabref@5.6.76/org.jabref.gui.util.DefaultTaskExecutor.convertToException(Unknown Source)
	at org.jabref@5.6.76/org.jabref.gui.util.DefaultTaskExecutor.lambda$getJavaFXTask$5(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.CompositeEventHandler.dispatchBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventUtil.fireEventImpl(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventUtil.fireEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.event.Event.fireEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.concurrent.EventHelper.fireEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.concurrent.Task.fireEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.concurrent.Task.setState(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.concurrent.Task$TaskCallable.lambda$call$2(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.application.PlatformImpl.lambda$runLater$10(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.application.PlatformImpl.lambda$runLater$11(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.glass.ui.InvokeLaterDispatcher$Future.run(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.glass.ui.win.WinApplication._enterNestedEventLoopImpl(Native Method)
	at org.jabref.merged.module@5.6.76/com.sun.glass.ui.win.WinApplication._enterNestedEventLoop(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.glass.ui.Application.enterNestedEventLoop(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.glass.ui.EventLoop.enter(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.tk.quantum.QuantumToolkit.enterNestedEventLoop(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.stage.Stage.showAndWait(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.scene.control.HeavyweightDialog.showAndWait(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.scene.control.Dialog.showAndWait(Unknown Source)
	at org.jabref@5.6.76/org.jabref.gui.JabRefDialogService.showCustomDialogAndWait(Unknown Source)
	at org.jabref@5.6.76/org.jabref.gui.importer.fetcher.WebSearchPaneViewModel.search(Unknown Source)
	at org.jabref@5.6.76/org.jabref.gui.importer.fetcher.WebSearchPaneView.lambda$initialize$3(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.CompositeEventHandler.dispatchBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.CompositeEventDispatcher.dispatchBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventUtil.fireEventImpl(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventUtil.fireEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.event.Event.fireEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.scene.Node.fireEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.scene.control.Button.fire(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.scene.control.behavior.ButtonBehavior.mouseReleased(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.scene.control.inputmap.InputMap.handle(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.CompositeEventHandler$NormalEventHandlerRecord.handleBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.CompositeEventHandler.dispatchBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.CompositeEventDispatcher.dispatchBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventUtil.fireEventImpl(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventUtil.fireEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.event.Event.fireEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.scene.Scene$MouseHandler.process(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.scene.Scene.processMouseEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.scene.Scene$ScenePeerListener.mouseEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.tk.quantum.GlassViewEventHandler$MouseEventNotification.run(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.tk.quantum.GlassViewEventHandler$MouseEventNotification.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.tk.quantum.GlassViewEventHandler.lambda$handleMouseEvent$2(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.tk.quantum.QuantumToolkit.runWithoutRenderLock(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.tk.quantum.GlassViewEventHandler.handleMouseEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.glass.ui.View.handleMouseEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.glass.ui.View.notifyMouse(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
	at org.jabref.merged.module@5.6.76/com.sun.glass.ui.win.WinApplication.lambda$runLoop$3(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalAccessError
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Unknown Source)
	at java.base/java.lang.reflect.Constructor.newInstance(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinTask.getThrowableException(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinTask.reportException(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinTask.invoke(Unknown Source)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateParallel(Unknown Source)
	at java.base/java.util.stream.AbstractPipeline.evaluate(Unknown Source)
	at java.base/java.util.stream.ReferencePipeline.collect(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.importer.fetcher.CompositeSearchBasedFetcher.performSearch(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.importer.SearchBasedFetcher.performSearch(Unknown Source)
	at org.jabref@5.6.76/org.jabref.gui.importer.fetcher.WebSearchPaneViewModel.lambda$search$3(Unknown Source)
	at org.jabref@5.6.76/org.jabref.gui.util.BackgroundTask$1.call(Unknown Source)
	at org.jabref@5.6.76/org.jabref.gui.util.DefaultTaskExecutor$1.call(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.concurrent.Task$TaskCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	... 1 more
Caused by: java.lang.IllegalAccessError: superclass access check failed: class org.glassfish.jaxb.runtime.api.JAXBRIContext (in module org.jabref.merged.module) cannot access class jakarta.xml.bind.JAXBContext (in module jakarta.xml.bind) because module org.jabref.merged.module does not read module jakarta.xml.bind
	at java.base/java.lang.ClassLoader.defineClass1(Native Method)
	at java.base/java.lang.ClassLoader.defineClass(Unknown Source)
	at java.base/java.lang.ClassLoader.defineClass(Unknown Source)
	at java.base/java.security.SecureClassLoader.defineClass(Unknown Source)
	at java.base/jdk.internal.loader.BuiltinClassLoader.defineClass(Unknown Source)
	at java.base/jdk.internal.loader.BuiltinClassLoader.findClassInModuleOrNull(Unknown Source)
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClassOrNull(Unknown Source)
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(Unknown Source)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(Unknown Source)
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
	at java.base/java.lang.Class.getDeclaredMethods0(Native Method)
	at java.base/java.lang.Class.privateGetDeclaredMethods(Unknown Source)
	at java.base/java.lang.Class.getMethodsRecursive(Unknown Source)
	at java.base/java.lang.Class.getMethod0(Unknown Source)
	at java.base/java.lang.Class.getMethod(Unknown Source)
	at jakarta.xml.bind@3.0.1/jakarta.xml.bind.ContextFinder.newInstance(Unknown Source)
	at jakarta.xml.bind@3.0.1/jakarta.xml.bind.ContextFinder.newInstance(Unknown Source)
	at jakarta.xml.bind@3.0.1/jakarta.xml.bind.ContextFinder.find(Unknown Source)
	at jakarta.xml.bind@3.0.1/jakarta.xml.bind.JAXBContext.newInstance(Unknown Source)
	at jakarta.xml.bind@3.0.1/jakarta.xml.bind.JAXBContext.newInstance(Unknown Source)
	at jakarta.xml.bind@3.0.1/jakarta.xml.bind.JAXBContext.newInstance(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.importer.fileformat.MedlineImporter.initUmarshaller(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.importer.fileformat.MedlineImporter.unmarshallRoot(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.importer.fileformat.MedlineImporter.importDatabase(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.importer.fetcher.MedlineFetcher.fetchMedline(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.importer.fetcher.MedlineFetcher.performSearch(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.importer.fetcher.CompositeSearchBasedFetcher.lambda$performSearch$1(Unknown Source)
	at java.base/java.util.stream.ReferencePipeline$7$1.accept(Unknown Source)
	at java.base/java.util.HashMap$KeySpliterator.tryAdvance(Unknown Source)
	at java.base/java.util.stream.StreamSpliterators$WrappingSpliterator.lambda$initPartialTraversalState$0(Unknown Source)
	at java.base/java.util.stream.StreamSpliterators$AbstractWrappingSpliterator.fillBuffer(Unknown Source)
	at java.base/java.util.stream.StreamSpliterators$AbstractWrappingSpliterator.doAdvance(Unknown Source)
	at java.base/java.util.stream.StreamSpliterators$WrappingSpliterator.tryAdvance(Unknown Source)
	at java.base/java.util.stream.StreamSpliterators$UnorderedSliceSpliterator$OfRef.forEachRemaining(Unknown Source)
	at java.base/java.util.stream.AbstractPipeline.copyInto(Unknown Source)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
	at java.base/java.util.stream.ReduceOps$ReduceTask.doLeaf(Unknown Source)
	at java.base/java.util.stream.ReduceOps$ReduceTask.doLeaf(Unknown Source)
	at java.base/java.util.stream.AbstractTask.compute(Unknown Source)
	at java.base/java.util.concurrent.CountedCompleter.exec(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinPool.scan(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(Unknown Source)
```

Some archives and portals are working.
- Sao/NASA ADS search took ages. I did NOT cancel, and it eventually found some stuff.

The non-working ones are these:

- ArXiv search took ages. I canceled the search after two minutes or so.
- DBLP search took ages. I canceled the search after four minutes or so.
- GVK search took ages. I canceled the search after three minutes or so.
- MathSciNet led to following error message:
   ```
   org.jabref.logic.importer.FetcherException: A network error occurred while fetching from https://mathscinet.ams.org/mathscinet/search/publications.html?pg7=ALLF&s7=critical%2C+engaged+and+change-oriented&r=1&extend=1&fmt=bibtex
	at org.jabref@5.6.76/org.jabref.logic.importer.SearchBasedParserFetcher.getBibEntries(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.importer.SearchBasedParserFetcher.performSearch(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.importer.SearchBasedFetcher.performSearch(Unknown Source)
	at org.jabref@5.6.76/org.jabref.gui.importer.fetcher.WebSearchPaneViewModel.lambda$search$3(Unknown Source)
	at org.jabref@5.6.76/org.jabref.gui.util.BackgroundTask$1.call(Unknown Source)
	at org.jabref@5.6.76/org.jabref.gui.util.DefaultTaskExecutor$1.call(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.concurrent.Task$TaskCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
   Caused by: java.io.IOException: Server returned HTTP response code: 401 for URL: https://mathscinet.ams.org/mathscinet/search/publications.html?pg7=ALLF&s7=critical%2C+engaged+and+change-oriented&r=1&extend=1&fmt=bibtex
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Unknown Source)
	at java.base/java.lang.reflect.Constructor.newInstance(Unknown Source)
	at java.base/sun.net.www.protocol.http.HttpURLConnection$10.run(Unknown Source)
	at java.base/sun.net.www.protocol.http.HttpURLConnection$10.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getChainedException(Unknown Source)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(Unknown Source)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(Unknown Source)
	at java.base/sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.net.URLDownload.asInputStream(Unknown Source)
	... 13 more
   Caused by: java.io.IOException: Server returned HTTP response code: 401 for URL: https://mathscinet.ams.org/mathscinet/search/publications.html?pg7=ALLF&s7=critical%2C+engaged+and+change-oriented&r=1&extend=1&fmt=bibtex
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(Unknown Source)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(Unknown Source)
	at java.base/java.net.HttpURLConnection.getResponseCode(Unknown Source)
	at java.base/sun.net.www.protocol.https.HttpsURLConnectionImpl.getResponseCode(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.net.URLDownload.openConnection(Unknown Source)
	... 14 more

   ```
- Medline/Pubmed led to following error message:
   ```
   java.lang.Exception: java.lang.IllegalAccessError: superclass access check failed: class org.glassfish.jaxb.runtime.api.JAXBRIContext (in module org.jabref.merged.module) cannot access class jakarta.xml.bind.JAXBContext (in module jakarta.xml.bind) because module org.jabref.merged.module does not read module jakarta.xml.bind
	at org.jabref@5.6.76/org.jabref.gui.util.DefaultTaskExecutor.convertToException(Unknown Source)
	at org.jabref@5.6.76/org.jabref.gui.util.DefaultTaskExecutor.lambda$getJavaFXTask$5(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.CompositeEventHandler.dispatchBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventUtil.fireEventImpl(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventUtil.fireEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.event.Event.fireEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.concurrent.EventHelper.fireEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.concurrent.Task.fireEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.concurrent.Task.setState(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.concurrent.Task$TaskCallable.lambda$call$2(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.application.PlatformImpl.lambda$runLater$10(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.application.PlatformImpl.lambda$runLater$11(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.glass.ui.InvokeLaterDispatcher$Future.run(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.glass.ui.win.WinApplication._enterNestedEventLoopImpl(Native Method)
	at org.jabref.merged.module@5.6.76/com.sun.glass.ui.win.WinApplication._enterNestedEventLoop(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.glass.ui.Application.enterNestedEventLoop(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.glass.ui.EventLoop.enter(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.tk.quantum.QuantumToolkit.enterNestedEventLoop(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.stage.Stage.showAndWait(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.scene.control.HeavyweightDialog.showAndWait(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.scene.control.Dialog.showAndWait(Unknown Source)
	at org.jabref@5.6.76/org.jabref.gui.JabRefDialogService.showCustomDialogAndWait(Unknown Source)
	at org.jabref@5.6.76/org.jabref.gui.importer.fetcher.WebSearchPaneViewModel.search(Unknown Source)
	at org.jabref@5.6.76/org.jabref.gui.importer.fetcher.WebSearchPaneView.lambda$initialize$3(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.CompositeEventHandler.dispatchBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.CompositeEventDispatcher.dispatchBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventUtil.fireEventImpl(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventUtil.fireEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.event.Event.fireEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.scene.Node.fireEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.scene.control.Button.fire(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.scene.control.behavior.ButtonBehavior.mouseReleased(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.scene.control.inputmap.InputMap.handle(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.CompositeEventHandler$NormalEventHandlerRecord.handleBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.CompositeEventHandler.dispatchBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.CompositeEventDispatcher.dispatchBubblingEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventUtil.fireEventImpl(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.event.EventUtil.fireEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.event.Event.fireEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.scene.Scene$MouseHandler.process(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.scene.Scene.processMouseEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.scene.Scene$ScenePeerListener.mouseEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.tk.quantum.GlassViewEventHandler$MouseEventNotification.run(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.tk.quantum.GlassViewEventHandler$MouseEventNotification.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.tk.quantum.GlassViewEventHandler.lambda$handleMouseEvent$2(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.tk.quantum.QuantumToolkit.runWithoutRenderLock(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.javafx.tk.quantum.GlassViewEventHandler.handleMouseEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.glass.ui.View.handleMouseEvent(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.glass.ui.View.notifyMouse(Unknown Source)
	at org.jabref.merged.module@5.6.76/com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
	at org.jabref.merged.module@5.6.76/com.sun.glass.ui.win.WinApplication.lambda$runLoop$3(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
   Caused by: java.lang.IllegalAccessError: superclass access check failed: class org.glassfish.jaxb.runtime.api.JAXBRIContext (in module org.jabref.merged.module) cannot access class jakarta.xml.bind.JAXBContext (in module jakarta.xml.bind) because module org.jabref.merged.module does not read module jakarta.xml.bind
	at java.base/java.lang.ClassLoader.defineClass1(Native Method)
	at java.base/java.lang.ClassLoader.defineClass(Unknown Source)
	at java.base/java.lang.ClassLoader.defineClass(Unknown Source)
	at java.base/java.security.SecureClassLoader.defineClass(Unknown Source)
	at java.base/jdk.internal.loader.BuiltinClassLoader.defineClass(Unknown Source)
	at java.base/jdk.internal.loader.BuiltinClassLoader.findClassInModuleOrNull(Unknown Source)
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClassOrNull(Unknown Source)
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(Unknown Source)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(Unknown Source)
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
	at java.base/java.lang.Class.getDeclaredMethods0(Native Method)
	at java.base/java.lang.Class.privateGetDeclaredMethods(Unknown Source)
	at java.base/java.lang.Class.getMethodsRecursive(Unknown Source)
	at java.base/java.lang.Class.getMethod0(Unknown Source)
	at java.base/java.lang.Class.getMethod(Unknown Source)
	at jakarta.xml.bind@3.0.1/jakarta.xml.bind.ContextFinder.newInstance(Unknown Source)
	at jakarta.xml.bind@3.0.1/jakarta.xml.bind.ContextFinder.newInstance(Unknown Source)
	at jakarta.xml.bind@3.0.1/jakarta.xml.bind.ContextFinder.find(Unknown Source)
	at jakarta.xml.bind@3.0.1/jakarta.xml.bind.JAXBContext.newInstance(Unknown Source)
	at jakarta.xml.bind@3.0.1/jakarta.xml.bind.JAXBContext.newInstance(Unknown Source)
	at jakarta.xml.bind@3.0.1/jakarta.xml.bind.JAXBContext.newInstance(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.importer.fileformat.MedlineImporter.initUmarshaller(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.importer.fileformat.MedlineImporter.unmarshallRoot(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.importer.fileformat.MedlineImporter.importDatabase(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.importer.fetcher.MedlineFetcher.fetchMedline(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.importer.fetcher.MedlineFetcher.performSearch(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.importer.SearchBasedFetcher.performSearch(Unknown Source)
	at org.jabref@5.6.76/org.jabref.gui.importer.fetcher.WebSearchPaneViewModel.lambda$search$3(Unknown Source)
	at org.jabref@5.6.76/org.jabref.gui.util.BackgroundTask$1.call(Unknown Source)
	at org.jabref@5.6.76/org.jabref.gui.util.DefaultTaskExecutor$1.call(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.concurrent.Task$TaskCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	... 1 more
   ```
- Springer had following error message:
```
org.jabref.logic.importer.FetcherException: A network error occurred while fetching from http://api.springernature.com/meta/v1/json?q=critical%2C+AND+engaged+AND+and+AND+change-oriented&api_key=a98b4a55181ffcd27259bea45edad12e&s=1&p=20
	at org.jabref@5.6.76/org.jabref.logic.importer.SearchBasedParserFetcher.getBibEntries(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.importer.SearchBasedParserFetcher.performSearch(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.importer.PagedSearchBasedParserFetcher.performSearch(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.importer.SearchBasedFetcher.performSearch(Unknown Source)
	at org.jabref@5.6.76/org.jabref.gui.importer.fetcher.WebSearchPaneViewModel.lambda$search$3(Unknown Source)
	at org.jabref@5.6.76/org.jabref.gui.util.BackgroundTask$1.call(Unknown Source)
	at org.jabref@5.6.76/org.jabref.gui.util.DefaultTaskExecutor$1.call(Unknown Source)
	at org.jabref.merged.module@5.6.76/javafx.concurrent.Task$TaskCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: Server returned HTTP response code: 403 for URL: http://api.springernature.com/meta/v1/json?q=critical%2C+AND+engaged+AND+and+AND+change-oriented&api_key=a98b4a55181ffcd27259bea45edad12e&s=1&p=20
	at java.base/jdk.internal.reflect.GeneratedConstructorAccessor49.newInstance(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Unknown Source)
	at java.base/java.lang.reflect.Constructor.newInstance(Unknown Source)
	at java.base/sun.net.www.protocol.http.HttpURLConnection$10.run(Unknown Source)
	at java.base/sun.net.www.protocol.http.HttpURLConnection$10.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getChainedException(Unknown Source)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(Unknown Source)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.net.URLDownload.asInputStream(Unknown Source)
	... 14 more
Caused by: java.io.IOException: Server returned HTTP response code: 403 for URL: http://api.springernature.com/meta/v1/json?q=critical%2C+AND+engaged+AND+and+AND+change-oriented&api_key=a98b4a55181ffcd27259bea45edad12e&s=1&p=20
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(Unknown Source)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(Unknown Source)
	at java.base/java.net.HttpURLConnection.getResponseCode(Unknown Source)
	at org.jabref@5.6.76/org.jabref.logic.net.URLDownload.openConnection(Unknown Source)
	... 15 more
```

- zbMath search took ages. I canceled after ca. 60 minutes or so.

--- a/build.gradle
+++ b/build.gradle
@@ -555,6 +555,7 @@ jlink {
         requires 'jdk.xml.dom'
         requires 'com.google.gson'
         requires 'org.slf4j'
+        requires 'jakarta.xml.bind'
         uses 'org.controlsfx.glyphfont.GlyphFont'
         uses 'com.airhacks.afterburner.views.ResourceLocator'
         uses 'com.airhacks.afterburner.injection.PresenterFactory'

--- a/src/main/java/module-info.java
+++ b/src/main/java/module-info.java
@@ -51,6 +51,8 @@ open module org.jabref {
     // Preferences and XML
     requires java.prefs;
     requires jakarta.xml.bind;
+    // needs to be loaded here as it's otherwise not found at runtime
+    requires org.glassfish.jaxb.runtime;
     requires jdk.xml.dom;
 
     // Annotations (@PostConstruct)

